### PR TITLE
Fix decision object schema namespaces

### PR DESCRIPTION
**Pull request by jbeckert**
Convert two existing schemas to non-UI, global dataset schemas and move their properties to the correct namespace

### CODE REVIEW

File: extensions/adobe/experience/offer-management/offer-placement.schema.json [hunk 1]
```diff
@@ -57,10 +66,13 @@
...
     {
-      "$ref": "#/definitions/offer-placement"
+      "$ref": "https://ns.adobe.com/xdm/data/record"
+    },
+    {
+      "$ref": "https://ns.adobe.com/experience/decisioning/placement"
     },
     {
-      "required": ["xdm:name", "xdm:channel", "xdm:componentType"]
+      "$ref": "#/definitions/offer-placement"
```
Comment (mjasrotia) [inline extensions/adobe/experience/offer-management/offer-placement.schema.json:75]: @jbeckert , This schema is already deprecated and continues to be so. You are still making changes to the allOF section here. Is there a need for this or am I missing something.

Comment (jbeckert) [reply extensions/adobe/experience/offer-management/offer-placement.schema.json:75]: It was a mistake to mark the status as deprecated. What needed to be deprecated were the "xdm:" properties in this schema

Comment (jbeckert) [reply extensions/adobe/experience/offer-management/offer-placement.schema.json:75]: Let me amend the schema and bring back the status to "stable"

Comment (mjasrotia) [reply extensions/adobe/experience/offer-management/offer-placement.schema.json:75]: Sounds good

**Review by mjasrotia — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**